src/tls/Makefile-tls.am

```diff
@@ -52,14 +52,25 @@ TEST_LDADD = \
 	$(NULL)
 
 TLS_TESTS = \
-	test-tls-connection				\
-	test-tls-server					\
+	test-tls-connection \
+	test-tls-server \
+	test-certfile \
```

Comment:
Maybe test-tls-certfile for consistency?

Reply:
It seemed redundant, but no strong feelings either way.

Reply:
Took your suggestion.